src/core/src/op/paged_attention.cpp

```diff
@@ -228,5 +296,10 @@ void PagedAttentionExtension::set_out_type(int index, const ov::element::Type& o
     m_output_type[index] = output_type;
 }
 
+const ov::element::Type PagedAttentionExtension::get_out_type(int index) const {
+    OPENVINO_ASSERT(index < 2, "Output index should be 0 or 1, but got " + std::to_string(index));
```

Comment:
```suggestion
    OPENVINO_ASSERT(index < 2, "Output index should be 0 or 1, but got " + std::to_string(index));
```
The `index < 2` will be also in error message maybe change also error message?
```suggestion
    OPENVINO_ASSERT(index < 2, "Output index should be 0 or 1, but got ", index);
```

Reply:
Done